test/typescript/store.ts

```diff
@@ -8,6 +8,8 @@ import {
   StoreEnhancerStoreCreator,
   Unsubscribe,
   Observer,
+  PreloadedState,
+  CombinedState
```

Comment:
This should make Prettier happy:
```suggestion
  CombinedState,
```